src/plugins/unified_search/public/utils/helpers.ts

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { IFieldType } from '@kbn/data-views-plugin/common';
+import { i18n } from '@kbn/i18n';
+import { isEmpty } from 'lodash';
+import { validateParams } from '../filter_bar/filter_editor/lib/filter_editor_utils';
+
+export const getFieldValidityAndErrorMessage = (
+  field: IFieldType,
+  value?: string | undefined
+): { isInvalid: boolean; errorMessage: string[] } => {
+  const type = field.type;
+  switch (type) {
+    case 'string':
```

Comment:
can we use values from the KBN_FIELD_TYPES ?

Reply:
Thanks for the comment. I will change to KBN_FIELD_TYPES.